Apps/AccountDataAccess/src/Patient/Api/IPatientIdentityApi.cs

```diff
@@ -0,0 +1,38 @@
+// -------------------------------------------------------------------------
+//  Copyright © 2019 Province of British Columbia
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// -------------------------------------------------------------------------
+namespace HealthGateway.AccountDataAccess.Patient.Api
+{
+    using System.Threading.Tasks;
+    using Refit;
+
+    /// <summary>
+    /// Interface to interact with PHSA Patient Identity API.
+    /// </summary>
+    internal interface IPatientIdentityApi
+    {
+        /// <summary>
+        /// Retrieves the patient by PID.
+        /// </summary>
+        /// <param name="hdid">The Hdid to lookup.</param>
+        /// <returns>The Patient matching the id.</returns>
+        [Get("/patient/{hdid}/patient-identity")]
+        Task<PatientIdentityResult?> PatientLookupByHdidAsync(string hdid);
```

Comment:
There's some inconsistency here.

the comments talk of a PID but he parameter is hdid, additionally, the patient-identity by hdid follows:

`/patient-identity/hdid/{hdid}`

![image](https://user-images.githubusercontent.com/19548348/235485686-7f1e089a-a5b6-4425-9213-21986d66b7d1.png)

Reply:
It is by Hdid.